src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Ball {
```

Comment:
Consider adding a constructor to initialize the Ball object with a colour and number. This will allow for the creation of Ball objects with specific states.